sonosco/inference/asr.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+
+from dataclasses import dataclass
+import torch.nn as nn
+
+
+@dataclass
+class SonoscoASR(ABC):
```

Comment:
I think it should also contain the decoder besides the model. Because these are the two defining components to do inference.

Reply:
Agree, however, I created a spearate task to handle the asr interface, not to mix it with ros. I'll create separate pr for this later.